docs/hub/timm.md

```diff
@@ -0,0 +1,162 @@
+# Using Timm at Hugging Face
+
+`timm`, also known as [pytorch-image-models](https://github.com/rwightman/pytorch-image-models), is an open-source collection of state-of-the-art PyTorch image models, pretrained weights, and utility scripts for training, inference, and validation.
+
+## Exploring Timm in the Hub
+
+You can find a number of `timm` models by filtering at the left of the [models page](https://huggingface.co/models?library=timm&sort=downloads).
+
+All models on the Hub come up with useful feature:
+1. An automatically generated model card, which model authors can complete with [information about their model](./models-cards).
+2. Metadata tags that help for discoverability, allowing users to find the model when searching for `timm` models.
+3. An [interactive widget](./models-widgets) you can use to play out with the model directly in the browser
+4. An [Inference API](./models-inference) that allows users to make inference requests
+
+## Using existing models
```

Comment:
Maybe update section header to say use existing models from the Hub or something like that? When I'm scanning the doc, it's not immediately clear that the model should be on the Hub and sounds like it can be from anywhere.